ssn/images/obs-foi-pattern.svg

```diff
@@ -1,4 +1,4 @@
x="497" y="1744" fill="light-dark(#000000, #ffffff)" font-family="&quot;Helvetica&quot;" font-size="15px" text-anchor="middle" font-style="italic">isSampleOf</text></switch></g></g></g><g data-cell-id="lf2BoHkVRqIrfnMeP8bU-74"><g><rect x="1" y="1" width="315" height="45" fill="#ffffff" stroke="#282c33" stroke-width="2.3" stroke-dasharray="16 6" pointer-events="all" style="fill: rgb(255, 255, 255); stroke: rgb(40, 44, 51);"/></g><g><g transform="translate(-0.5 -0.5)"><switch><foreignObject style="overflow: visible; text-align: left;" pointer-events="none" width="100%" height="100%" requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility"><div xmlns="http://www.w3.org/1999/xhtml" style="display: flex; align-items: unsafe center; justify-content: unsafe flex-start; width: 317px; height: 1px; padding-top: 24px; margin-left: 1px;"><div style="box-sizing: border-box; font-size: 0; text-align: left; width: 315px; max-height: 45px; color: #000000; "><div style="display: inline-block; font-size: 30px; font-family: &quot;Helvetica&quot;; color: light-dark(#000000, #ffffff); line-height: 1.2; pointer-events: all; width: 100%; white-space: normal; word-wrap: normal; ">Direct observation of ultimate feature</div></div></div></foreignObject><text x="1" y="33" fill="light-dark(#000000, #ffffff)" font-family="&quot;Helvetica&quot;" font-size="30px">Direct observation of...</text></switch></g></g></g><g data-cell-id="lf2BoHkVRqIrfnMeP8bU-75"><g><rect x="1" y="556" width="360" height="45" fill="#ffffff" stroke="#282c33" stroke-width="2.3" stroke-dasharray="16 6" pointer-events="all" style="fill: rgb(255, 255, 255); stroke: rgb(40, 44, 51);"/></g><g><g transform="translate(-0.5 -0.5)"><switch><foreignObject style="overflow: visible; text-align: left;" pointer-events="none" width="100%" height="100%" requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility"><div xmlns="http://www.w3.org/1999/xhtml" style="display: flex; align-items: unsafe center; justify-content: unsafe flex-start; width: 362px; height: 1px; padding-top: 579px; margin-left: 1px;"><div style="box-sizing: border-box; font-size: 0; text-align: left; width: 360px; max-height: 45px; color: #000000; "><div style="display: inline-block; font-size: 30px; font-family: &quot;Helvetica&quot;; color: light-dark(#000000, #ffffff); line-height: 1.2; pointer-events: all; width: 100%; white-space: normal; word-wrap: normal; ">Indirect observation using sample of ultimate feature</div></div></div></foreignObject><text x="1" y="588" fill="light-dark(#000000, #ffffff)" font-family="&quot;Helvetica&quot;" font-size="30px">Indirect observation usi...</text></switch></g></g></g></g></g></g><switch><g requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility"/><a transform="translate(0,-5)" xlink:href="https://www.drawio.com/doc/faq/svg-export-text-problems" target="_blank"><text text-anchor="middle" font-size="10px" x="50%" y="100%">Text is not SVG - cannot display</text></a></switch></svg>
```

Comment:
I can't narrow focus better than this... but there is at least one rectangle in this image that retains the white background.